seed/views/01002_region_caseload_view.js

```diff
@@ -0,0 +1,28 @@
+exports.seed = function (knex, Promise) {
```

Comment:
Can you merge in the latest `master`? The ordering on the naming of these files has changed

Reply:
merged and renamed the new views